src/style.css

```diff
@@ -1 +1,62 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  text-transform: uppercase;
+  list-style-type: none;
+  margin-right: 50px;
```

Comment:
instead making margin-right for nav list make horizontal paddings for header
pay attention on reference. list shouldn't have any horizontal spaces
![Screenshot from 2023-04-11 14-36-10](https://user-images.githubusercontent.com/33029864/231151718-84f60883-e8d1-423f-972b-484ba8ec40ca.png)